.github/actions/parallel-ctest-containers/main.mjs

```diff
@@ -12,7 +12,7 @@ const log_tarball_prefix = core.getInput('log-tarball-prefix', {required: true})
 const tests_label = core.getInput('tests-label', {required: true});
 
 try {
-   if(child_process.spawnSync("docker", ["run", "--name", "base", "-v", `${process.cwd()}/build.tar.zst:/build.tar.zst`, "--workdir", "/__w/leap/leap", container, "sh", "-c", "zstdcat /build.tar.zst | tar x"], {stdio:"inherit"}).status)
+   if(child_process.spawnSync("docker", ["run", "--privileged", "--name", "base", "-v", `${process.cwd()}/build.tar.zst:/build.tar.zst`, "--workdir", "/__w/leap/leap", container, "sh", "-c", "zstdcat /build.tar.zst | tar x"], {stdio:"inherit"}).status)
```

Comment:
you'll need to regenerate the [dist/index.mjs](https://github.com/AntelopeIO/leap/blob/main/.github/actions/parallel-ctest-containers/dist/index.mjs) it seems I failed to document this well anywhere; probably because I didn't think we'd still be using it still.

```
cd .github/actions/parallel-ctest-containers
npm ci
# stay in parallel-ctest-containers directory
./node_modules/.bin/ncc build main.mjs --minify
```

I _hope_ this ends up with an dist/index.mjs that has just this minimal change in it for you -- I really hope it regenerates this minified version reproducibly/deterministically for you. If it's a huge change maybe don't commit it: I kinda hate to continue to bloat the repo with this large file  which ultimately we'll be removing. Worst case we can manually modify the dist/index.mjs with this ~dozen character change if it comes to it.

Reply:
Due to the minification, the diff is very large.  Something more than half the file.  Should I commit it or try to unravel the minification?

Reply:
oof, that's a major bummer. I'm no JS dep expert, but I was hopeful that by having the package lock committed others would get the same versions of everything ergo the recreation of the amalgamation (minified or not) would be reproducible.

You can either
* just commit the big blob diff (kinda yucky), or
* just manually modify the one line in the big blob diff to add that `"--privileged", ` change. (not really so bad to just fine that one line in the file and change it, just a wee bit of work